### PR TITLE
Fix menu item splice CMS-39 #resolve

### PR DESCRIPTION
**Pull request by diutsu**

### CODE REVIEW

File: src/main/webapp/WEB-INF/fenixedu-cms/editPage.jsp [hunk 1]
```diff
@@ -595,7 +593,7 @@ ${portal.angularToolkit()}
                         var menuItem = findMenuItem();
                         var menuItemParent = findMenuItemParent(menuItem);
                         if(menuItem && menuItemParent && menuItemParent.children) {
-                            menuItemParent.children.splice(menuItemParent.children.indexOf(menuItem));
+                            menuItemParent.children.splice(menuItemParent.children.indexOf(menuItem), 1);
```
Comment (davidmartinho) [inline src/main/webapp/WEB-INF/fenixedu-cms/editPage.jsp]: Splice Argument is Coming

Comment (sergiofbsilva) [inline src/main/webapp/WEB-INF/fenixedu-cms/editPage.jsp]: lost menu

@nurv

Comment (diutsu) [inline src/main/webapp/WEB-INF/fenixedu-cms/editPage.jsp]: lost menu on checkbox

Comment (Paulo-Branco) [inline src/main/webapp/WEB-INF/fenixedu-cms/editPage.jsp]: Aint nobody got time for that

**Pull request merged**